www/front_src/src/Authentication/Openid/models.ts

```diff
@@ -18,14 +28,19 @@ export interface OpenidConfiguration {
 }
 
 export interface OpenidConfigurationToAPI {
```

Comment:
why are there 2 cases? 
`OpenidConfiguration` & `OpenidConfigurationToAPI`

One for the Frontend components and a special one for the communication with the API?
Only 2 properties are different, and mandatory compared to the other is it?

Reply:
It's just about cases because we are using camel case but backend is using snake case so the only way is to create an interface that match with the backend properties case

Reply:
Ok, thanks for your answer :)